src/components/WalletDropdown/index.tsx

```diff
@@ -36,13 +36,13 @@ const WalletDropdownWrapper = styled.div`
   position: absolute;
   top: 65px;
   right: 20px;
+  z-index: 2;
```

Comment:
we have a `Z_INDEX` enum in the theme folder - i'm okay adding 2 in this case for it as a deprecated value if the other refactor would be a lot of work, but in the future hoping to move everything there 🤞